csharp-linq/SystemDynamicLinqCoreLibrary/Tests/SystemDynamicLinqCoreLibraryUnitTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Linq.Dynamic.Core;
+using System.Linq.Expressions;
+using SystemDynamicLinqCoreLibrary;
+
+namespace Tests
+{
+    public class SystemDynamicLinqCoreLibraryUnitTests
+    {
+        private Employee _employee;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _employee = new Employee();
+        }
```

Comment:
While these tests are fine, you're not testing the application. We have a lot of code in the program.cs and we don't know if it works or not, without running the application. You need to test this code instead of recreating the Employee class in the test project and testing it separately.

Please create classes with methods that demonstrate different examples, and then test these instead of creating new ones.

Reply:
I have updated the application by creating an additional TestData class. The class has methods demonstrating the examples, all tested. 
Please check the latest commits.

Reply:
Much better, thanks!